src/test/java/org/zalando/nakadi/repository/kafka/NakadiKafkaConsumerTest.java

```diff
@@ -98,18 +98,18 @@ public void whenCreateConsumerThenKafkaConsumerConfiguredCorrectly() {
     public void whenReadEventsThenGetRightEvents() {
 
         // ARRANGE //
-        final String event1 = randomString();
-        final String event2 = randomString();
+        final byte[] event1 = randomString().getBytes();
+        final byte[] event2 = randomString().getBytes();
```

Comment:
@lhotari did you push the getBytes() changes? Just in case my view is stale, I'm looking at 5fd783da9b32a18693d052a4062d870f59baa007 as the latest and it's still showing bare getBytes()